lib/import.js

```diff
@@ -6,16 +6,17 @@ const {Preprocessor, VERSION, GRAMMAR_VERSION} = require('./preprocessor');
 const {DataElementImporter} = require('./dataElementListener');
 const {ValueSetImporter} = require('./valueSetListener');
 const {MappingImporter} = require('./mappingListener');
-const {CimcoreImporter} = require('./cimcore/cimcoreImport');
+//const {CimcoreImporter} = require('./cimcore/cimcoreImport');
```

Comment:
Commented out everything CIMCORE. Denigrated, not using it in the tests. The tests are all broken because they relied on very old CIMPL that was hand-rendered into CIMCORE JSON.

Reply:
OK -- but since you commented it out in the importer itself (not the tests), this means that the SHR CLI no longer supports importing CIMCORE.  And if that's the case, we need to update the CLI and its documentation to reflect that.

So to confirm -- are you saying that importing CIMCORE is no longer a requirement/feature of SHR CLI?